src/Common/src/Interop/Unix/libcurl/Interop.libcurl_slist.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+using size_t = System.UInt64;
```

Comment:
Seeing size_t declared as UInt64 in some files and IntPtr in others is making me nervous. Do we test (or support) both bitnesses on Unix?